src/BizHawk.Client.Common/movie/PlatformFrameRates.cs

```diff
@@ -17,6 +17,7 @@ public static class PlatformFrameRates
 
 		private static readonly Dictionary<string, double> Rates = new Dictionary<string, double>
 		{
+			["Panasonic3DO"] = 60.0, // The emulator (Opera-Libretro) reports exact 60.0 for NTSC https://github.com/libretro/opera-libretro/blob/67a29e60a4d194b675c9272b21b61eaa022f3ba3/libopera/opera_region.c#L10

```

Comment:
Please add the PAL variant too (same key but with a `"_PAL"` suffix).

Reply:
added